Cargo.toml

```diff
@@ -1,10 +1,11 @@
 [workspace]
 members = [
     "rust/template",
+    "rust/dyn-ddlog",
```

Comment:
Before this sticks, I wonder if `ddlog` is a better name. After all, this is meant as _the_ new DDlog compiler, not just a dynamic version of it.